### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by mehedihassan0**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by realboy918** [comment]
ok

**Review by realboy918 — COMMENTED** [review]
Keep things more beautiful

**Review by mehedihassan0 — COMMENTED** [review]
ok i have reviewed

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (mehedihassan0) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (mehedihassan0) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again  and again🧑‍💻 🤖!</h2>
```

**Pull request merged**